docs/getting-started.asciidoc

```diff
@@ -0,0 +1,224 @@
+unchecked. Specify `shakes*`  as the index pattern for the Shakespeare data set and click *Create* to define the index 
+pattern, then define a second index pattern named `ba*`.
+
+[float]
+[[tutorial-discovering]]
+=== Discovering Your Data
+
+Click the *Discover* tab to display Kibana's data discovery functions:
+
+image::images/tutorial-discover.png[]
+
+Right under the tab itself, there is a search box where you can search your data. Searches take a specific 
+{ref}/query-dsl-query-string-query.html#query-string-syntax[query syntax] that enable you to create custom searches, 
+which you can save and load by clicking the buttons to the right of the search box.
+
+Beneath the search box, the current index pattern is displayed in a drop-down. You can change the index pattern by 
+selecting a different pattern from the drop-down selector.
+
+Try selecting the `ba*` index pattern and putting the following search into the search box:
+
+[source,text]
+account_number:<100 AND balance:>47500
+
+If you're using the linked sample data set, this search returns 5 results: Account numbers 8, 32, 78, 85, and 97.
+
+image::images/tutorial-discover-2.png[]
+
+To narrow the display to only the specific fields of interest, highlight each field in the list that displays under the 
+index pattern and click the *Add* button. Note how, in this example, adding the `account_number` field changes the 
+display from the full text of five records to a simple list of five account numbers:
+
+image::images/tutorial-discover-3.png[]
+
+[float]
+[[tutorial-visualizing]]
+=== Data Visualization: Beyond Discovery
+
+The visualization tools available on the *Visualize* tab enable you to display aspects of your data sets in several 
+different ways. Visualizations depend on Elasticsearch {ref}/search-aggregations.html[aggregations] in two different 
+types: _bucket_ aggregations and _metric_ aggregations. A bucket aggregation sorts your data according to criteria you 
+specify. For example, in our accounts data set, we can establish a range of account balances, then display what 
+proportions of the total fall into which range of balances.
+
+Click on the *Visualize* tab to start:
+
+image::images/tutorial-visualize.png[]
+
+Click on *Pie chart*, then *From a new search*. Select the `ba*` index pattern. The whole pie displays, since we 
+haven't specified any buckets yet.
+
+image::images/tutorial-visualize-pie-1.png[]
+
+Select *Split Slices* from the *Select buckets type* list, then select *Range* from the *Aggregation* drop-down 
+selector. Select the *balance* field from the *Field* drop-down, then click on *Add Range* four times to bring the 
+total number of ranges to six. Enter the following ranges:
+
+[source,text]
+0            1000
+1000         3000
+3000         7000
+7000        15000
+15000       31000
+31000       50000
+
+Click the green *Apply changes* to display the chart:
+
+image::images/tutorial-visualize-pie-2.png[]
```

Comment:
This screenshot has the builder collapsed. Expanding it would allow the user to follow along easier